usefulUtilities/botPlayer/README.md

```diff
@@ -0,0 +1,14 @@
+# Bot Player
+Bot Player helps with domain testing bots that need to be placed in a certain location, are a certain unique set, and that need to last through domain resets.
+
+## Setup
+1. use the botRecorder.js to record the bots that you need.  When you click on the tablet button, it will start recording and then stop after you click it again. 
+2. Keep the script update up to check on the status and to see where the file was saved.
```

Comment:
what is the "script update"?

Reply:
ahh, I most have zoned out while writing that.  I meant the script HMD log lol.

Reply:
done